### PR TITLE
AST-1914 - fix: The off-canvas dropdown menu arrow is misplaced

### PR DESCRIPTION
**Pull request by yssambare12**
*Issue:*
The off-canvas dropdown menu arrow is misplaced(in the middle) rather than left and the arrow is taking some weird space.

However, it seems to occur with the latest version(101) of the Chrome browser and not occurring with the older versions(Chrome 96).

*Steps to Reproduce:*

Go to the Customizer>Switch to mobile view>Off-canvas setting>Content Alignment>Right. After that visit the front-end mobile view.

*Screencast:*

https://d.pr/v/rfmnGr

Slack conversation:https://brainstormforce.slack.com/archives/CDS7TQ75Z/p1651991763290919

*Ticket URL:*

https://secure.helpscout.net/conversation/1871517102/183017/
### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php [hunk 1]
```diff
@@ -541,7 +541,8 @@ function astra_dropdown_type_static_css() {
 		$dropdown_type_css .= '
 		.ast-mobile-header-content.content-align-flex-end .main-header-bar-navigation .menu-item-has-children > .ast-menu-toggle,
 		.ast-desktop-header-content.content-align-flex-end .main-header-bar-navigation .menu-item-has-children > .ast-menu-toggle {
-		  	left: calc( 20px - 0.907em);
```
Comment (GauravKhupse) [inline inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php:544]: Add changelog llne for this PR

File: inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php [hunk 2]
```diff
@@ -541,7 +541,8 @@ function astra_dropdown_type_static_css() {
...
+			left: calc( 20px - 0.907em);
+			display: flex;
```
Comment (nathanrodrigues2111) [inline inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php:545]:
```suggestion
			right: auto;
```

Comment (nathanrodrigues2111) [reply inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php:545]: Adding flex may cause issue. Please change it to right: auto

Comment (yssambare12) [reply inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php]: Updated & also added a fix for RTL as well.

**Review by GauravKhupse — APPROVED** [review]

File: inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php [hunk 3]
```diff
@@ -489,6 +489,7 @@ function astra_dropdown_type_static_css() {
 		.ast-mobile-header-content.content-align-flex-end .main-header-bar-navigation .menu-item-has-children > .ast-menu-toggle,
 		.ast-desktop-header-content.content-align-flex-end .main-header-bar-navigation .menu-item-has-children > .ast-menu-toggle {
 		  	right: calc( 20px - 0.907em);
+			right: auto;
```
Comment (nathanrodrigues2111) [inline inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php:492]:
```suggestion
			left: auto;
```

Comment (yssambare12) [reply inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php]: Updated.

**Review by imnavanath — APPROVED** [review]

**Pull request merged**